### ISSUE TITLE
TypeError in trulens: issubclass() Argument Must Be a Class When Wrapping RAG with TruCustomApp

### ISSUE THREAD

**Issue by gryhkn** [truera/trulens#775]
Hello,

I've encountered a `TypeError` while following the Quickstart guide on the TruLens documentation page (specifically the "Construct the app" section found at https://www.trulens.org/trulens_eval/quickstart/).

Here's the code snippet that is causing the issue:

```python
from trulens_eval import TruCustomApp

tru_rag = TruCustomApp(rag,
                       app_id='RAG v1',
                       feedbacks=[f_groundedness, f_qa_relevance, f_context_relevance])
```

However, when I run this snippet, I receive the following error message:

```
TypeError: issubclass() arg 1 must be a class
```

I have followed the installation steps exactly as described on the Quickstart page, including the necessary imports and environment setup. The error occurs when trying to instantiate TruCustomApp with my RAG model.

Could you please help me understand what might be going wrong here? Am I missing a step, or could there be an issue with the TruCustomApp class definition?

For reference, I'm using the following package versions:

```
trulens_eval: 0.2.0
chroma_db: 4.18
openai: 1.3.7
```

Any guidance on this would be greatly appreciated. Thank you for your support.

**Comment by joshreini1** [comment]
Hey @gryhkn - thanks for reporting. This is fixed by https://github.com/truera/trulens/pull/770 and will be available in the next release.

For instructions on installing from this branch, see:
https://www.trulens.org/welcome/#installation-and-setup

**Comment by ZS4MLDL** [comment]
I tried installing branch https://github.com/truera/trulens/tree/piotrm/optionals_and_subclass_fixes, however not able to run it successfully.

Please find the error trace

File "c:\Code\tru_lens\feedback_func.py", line 1, in <module>
    from trulens_eval import Feedback, Select
  File "C:\Code\trulen_env\lib\site-packages\trulens_eval\__init__.py", line 83, in <module>
    from trulens_eval.feedback import Bedrock
  File "C:\Code\trulen_env\lib\site-packages\trulens_eval\feedback\__init__.py", line 17, in <module>
    from trulens_eval.feedback.groundedness import Groundedness
  File "C:\Code\trulen_env\lib\site-packages\trulens_eval\feedback\groundedness.py", line 11, in <module>
    from trulens_eval.feedback.provider.litellm import LiteLLM
  File "C:\Code\trulen_env\lib\site-packages\trulens_eval\feedback\provider\litellm.py", line 11, in <module>
    import litellm
ModuleNotFoundError: No module named 'litellm'

**Comment by joshreini1** [comment]
Hey @ZS4MLDL @gryhkn - these issues should be fixed in the new release, 0.20.2 available now. Please upgrade and report back if these issues continue. Closing for now.

**Comment by ZS4MLDL** [comment]
I tried installing  0.20.2, however I was getting same error. Finally, I had to install litellm and boto3 to make it working..

pip install litellm
pip install boto3
